ibm_catalog.json

```diff
@@ -287,189 +287,15 @@
             },
             {
               "key": "default_worker_pool_machine_type",
+              "type": "string",
+              "description": "A worker node flavor that defines the architecture, virtual CPU, memory, and disk space in the default worker pool.",
```

Comment:
please do not add type or description here. They are already set in the terraform code, so it will be pulled from there if not set. Otherwise you have 2 places to maintain